targets/mgmarkdownfmt/targets.go

```diff
@@ -0,0 +1,42 @@
+package mgmarkdownfmt
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/go-logr/logr"
+	"github.com/magefile/mage/mg"
+	"github.com/magefile/mage/sh"
+	"go.einride.tech/mage-tools/mglog"
+	"go.einride.tech/mage-tools/mgtool"
+)
+
+const version = "75134924a9fd3335f76a9709314c5f5cef4d6ddc"
+
+var executable string
+
+func FormatMarkdown(ctx context.Context) error {
+	ctx = logr.NewContext(ctx, mglog.Logger("format-markdown"))
+	logr.FromContextOrDiscard(ctx).Info("formatting...")
+	mg.Deps(prepare)
+	return sh.RunV(executable, "-w", ".")
+}
+
+func prepare() error {
+	const toolName = "markdownfmt"
+	toolDir := filepath.Join(mgtool.GetPath(), toolName, version)
+	executable = filepath.Join(toolDir, toolName)
+	if _, err := os.Stat(executable); err == nil {
+		return nil
+	}
+	return sh.RunWithV(
```

Comment:
Please use https://github.com/einride/mage-tools/blob/b17068b60126a0460b2c31a1dafa1745a9c2c37d/mgtool/build.go#L13 `GoInstall` instead :) here is an example on when we use it https://github.com/einride/mage-tools/blob/main/targets/mggolangmigrate/targets.go#L25

Reply:
Fixed!